pkg/kubelet/volumes.go

```diff
@@ -132,8 +147,9 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 			glog.Errorf("Could not create volume mounter for pod %s: %v", pod.UID, err)
 			return nil, err
 		}
+		//This should never be hit if there is a failure in mounter - probably can remove??
```

Comment:
I think it is reachable unless I am missing something.
`newVolumeMounterFromPlugins` can return `nil, nil` although `FindPluginBySpec` which it calls never returns `nil, nil`
I think it is safe to remove but might as well leave it for completeness. Eitherway I would remove the comment above